### DISCUSSION TITLE
Tooltip Content

### DISCUSSION THREAD

**Discussion by GamingInfinite** [saadeghi/daisyui#1575] [Ideas / feature requests]
For a majority of use cases the tooltip feature works.  But I started a project where I basically want to be able to add more content than just text to a tooltip, for instance, show a different webpage using an <iframe> element when hovering over it.  If there is a way to do this with the current implementation I'm all ears, but if there isn't I feel like making it more robust in this way gives it way more flexibility

**Comment by saadeghi** [comment]
It's not possible with daisyUI tooltip because daisyUI tooltip works with data attribute. Data attribute can only contain text.

You might want to use [`dropdown-hover`](https://daisyui.com/components/dropdown/#dropdown-on-hover)  
Or you can use a JS plugin for hover.
